src/testing/kuttl/oneagent/oneagent-test.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
 startKIND: false
-kindConfig: kind.yaml
+kindConfig: src/testing/kuttl/kind.yaml
 skipDelete: false
+skipClusterDelete: true
```

Comment:
it's better to delete cluster here than in Makefile

Reply:
According to docs, this flag only skips deleting a mocked control plane or KIND clusters. The former doesn't matter since it's a mock, the latter should be discarded after a test run anyway. So I'd agree with @mjgrzybek here.

https://kuttl.dev/docs/testing/reference.html#testsuite

Reply:
Agree